plugins/module_utils/kibana.py

```diff
@@ -17,6 +17,7 @@
 from urllib.error import HTTPError
 import urllib.parse
 
+
 class Kibana(object):
```

Comment:
no need to explicitly inherit object for your classes in python 3, "class Kibana:" alone should be fine